releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,6 +1,13 @@
 # Microsoft.Extensions.Configuration.AzureAppConfiguration
 [Source code][source_code] | [Package (NuGet)][package]
 
+## 7.3.0 - July 16th, 2024
+### Bug Fixes
+* Fixed a bug where the provider would throw instead of handling a `FormatException` when the optional parameter was set to `true` in the `AddAzureAppConfiguration` method, or when calling `TryRefreshAsync`. [#551](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/551)
```

Comment:
Maybe we can skip implementation details:

### Bug Fixes: 

Loading invalid feature flags should not throw an error when 'TryRefreshAsync' is invoked or when 'optional' argument is true in 'AddAzureAppConfiguration' method.

Reply:
Yea that sounds better, I just wasn't sure how to phrase it since I thought we were throwing format exception from other places too like [invalid connection string](https://github.com/Azure/AppConfiguration-DotnetProvider/blob/0c9e17a9e04b7be98388db7c7dc6e0dffc37939e/src/Microsoft.Extensions.Configuration.AzureAppConfiguration/ConnectionStringUtils.cs#L32) on startup. I don't think we have to mention it though?